package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-data-table-component",
-  "version": "7.4.7",
+  "version": "7.4.8",
```

Comment:
This is a new feature so it should be 7.5.0

Reply:
One last comment and this should be GTG

Reply:
done